commcare_export/checkpoint.py

```diff
@@ -462,6 +472,9 @@ def get_pagination_mode(self, checkpoint_manager):
         if self.start_over or self.since or not checkpoint_manager:
             return PaginationMode.date_indexed
 
+        if checkpoint_manager.data_source == 'ucr':
```

Comment:
should this go above the previous `if` statement?

Reply:
@snopoke I think technically you're right. I initially thought of doing that, but then tested it and it seemed fine, although I think my testing might not have included all the scenarios, so I'll move it.